src/messages/es.json

```diff
@@ -226,16 +320,16 @@
   "support_status.title": "Status",
   "support_status.description": "Vea el registro del historial de la plataforma.",
   "support_plans.title": "Planes de soporte",
-  "support_plans.link": "/faq/como-funciona-el-soporte-de-vtex",
   "support_plans.description": "Consulte los detalles de los planes de soporte disponibles para cada plan comercial.",
+  "announcement_bar.headline": "📢 Queremos saber más sobre ti y cómo utilizas nuestra documentación. ",
+  "announcement_bar.button": "¡Complete nuestra encuesta! Toma menos de 5 minutos.",
```

Comment:
this was removed before. why add it back?

Reply:
Maybe the branch was created before we merged this removal. Anyway, I agree this should be removed.

Reply:
Exactly! Removed!